governor/gov.gno

```diff
@@ -0,0 +1,453 @@
+package governor
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/binary"
+	"std"
+	"time"
+
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/ufmt"
+	u256 "gno.land/p/demo/uint256"
+)
+
+var (
+	timeNow   = time.Now
+	gGovernor *Governor
+)
+
+// Init initializes the governor contract with the given staker and configuration.
+// It creates a new Governor instance and sets it as the global governor.
+// This function should be called only once at the beginning of the contract's lifecycle.
+func Init(staker IStakerDispatcher, config Config) {
```

Comment:
Does it have to call manually?
Can't we just init them in `init()`

Reply:
Using `init()` is quite hard to initialize multiple times or apply different settings, so I decided to do it manually for flexibility.